lib/features/auto-resize/AutoResize.js

```diff
@@ -245,4 +246,45 @@ function boundsChanged(newBounds, oldBounds) {
     newBounds.width !== oldBounds.width ||
     newBounds.height !== oldBounds.height
   );
+}
+
+/**
```

Comment:
As a general concern, could we simplify the jsdoc statements to either describe the arguments with substantial hints _or_ remove the description all together?

The description "- Old bounds." for a param `@param {Object} oldBounds` does not contribute to my understanding of the code. 

It would, however contribute to my understanding if the type was `Bounds`, not `Object`.

Reply:
If you agree with my comment we could probably remove all occurences of `/ - [^.]+\./` from the code base :wink: